.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out branch
+        uses: actions/checkout@v3
+
+      - name: Setup NodeJS 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      # Callling `npm i` because:
+      # `npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync

```

Comment:
It's odd that the files are out of sync. Usually a simple npm install followed by commit to the lock are enough to resolve the problem. Can we try that and switch to npm ci now?